tools/git.md

```diff
@@ -29,6 +29,13 @@ we want to collect - e.g. the different ways of identifying a commit
 - tags, branches, HEAD~2, hash, short hash 0 for use with the git diff 
 etc.
 
+### Push to 2 remotes at once 
```

Comment:
This should be https://github.com/OpenDataServices/developer-docs/issues/38 not https://github.com/OpenDataServices/developer-docs/issues/37